web/src/styles/global.css

```diff
@@ -4,5 +4,8 @@
 
 * {
     font-family: 'Montserrat', sans-serif;
+}
+
+body {
     background-color: black;
```

Comment:
I know only part of this is part of this pr, but I don't think we want/need to declare a background color. Are the majority of the components going to have a black background?

Reply:
@asasmith no I will remove it, its just for development so I can see things until i make the first component, which I am working on now.